src/Umbraco.Web/Editors/Filters/MemberValidationHelper.cs

```diff
@@ -117,8 +118,9 @@ public override bool ValidateProperties(MemberSave model, IContentProperties<Con
             //if a sensitive value is being submitted.
             if (UmbracoContextAccessor.UmbracoContext.Security.CurrentUser.HasAccessToSensitiveData() == false)
             {
-                var sensitiveProperties = model.PersistedContent.ContentType
-                    .PropertyTypes.Where(x => model.PersistedContent.ContentType.IsSensitiveProperty(x.Alias))
+                var contentType = Current.Services.MemberTypeService.Get(model.PersistedContent.ContentTypeId);
```

Comment:
The `MemberValidationHelper` has things injected into it so there's no reason to use Current here